cmd/manager/main.go

```diff
@@ -56,10 +54,12 @@ const (
 	CertValidityFlag       = "cert-validity"
 	CertRotateBeforeFlag   = "cert-rotate-before"
 
-	AutoInstallWebhooksFlag = "auto-install-webhooks"
-	OperatorNamespaceFlag   = "operator-namespace"
-	WebhookSecretFlag       = "webhook-secret"
-	WebhookPodsLabelFlag    = "webhook-pods-label"
+	OperatorNamespaceFlag = "operator-namespace"
+
+	SetupWebhookCertsFlag    = "setup-webhook-certs"
```

Comment:
Should this maybe be `manage-webhook-certs`? Since it does more than just initial setup